### ISSUE TITLE
Missing "GettingStarted" example?

### ISSUE THREAD

**Issue by jonesmz** [GameFoundry/bsfExamples#2]
Just to double check, did you want to drop that example entirely? I thought the dragon was a reasonable demo-app.

**Comment by BearishSun** [comment]
Yeah I thought I'd drop it and instead create smaller, more focused examples. PhysicallyBasedRendering example already takes care of the rendering portion, and I'd add another example for GUI, as well as a few others.

But I'm open to changing my mind if you think it's a good idea to keep it.

**Comment by jonesmz** [comment]
I think that more working examples don't hurt, so I'd vote to keep it just for that sake.

However, if you wanted to treat it as an example for camera manipulation, and increase the sophistication of the camera behavior, or perhaps something else, like a simple animation example, either of those would be distinct but still useful.
